### PR TITLE
Update Classical McEliece

### PR DESCRIPTION
**Pull request by TomerHawk**
It would be possible to add 5 more parameter Sets, which are also implemented and submitted in round 3 .

### CODE REVIEW

File: encryption/classic-mceliece/classic-mceliece.yaml [hunk 1]
```diff
@@ -18,14 +18,21 @@ authors:
   - Nicolas Sendrier
   - Jakub Szefer
   - Wen Wang
+  - Martin R. Albrecht
+  - Carlos Cid
+  - Jan Gilcher
+  - Varun Maram
+  - Kenneth G. Paterson
+  - Cen Jung Tjhai
+  - 'Martin Tomlinson,'
```
Comment (benj-zen) [inline encryption/classic-mceliece/classic-mceliece.yaml:27]: copy paste error

**Review by benj-zen — CHANGES_REQUESTED** [review]
Looks good. Just the one comment.

**Comment by benj-zen** [comment]
I'd also add the new parameter sets.

**Comment by TomerHawk** [comment]
> I'd also add the new parameter sets.

Okay, I will add them

**Review by benj-zen — APPROVED** [review]

**Pull request merged**